### PR TITLE
Resolved some memory leaks in unit tests

### PR DESCRIPTION
**Pull request by marcusnaslund**
Before:
```
definitely lost: 12,256 bytes in 486 blocks
indirectly lost: 54,795 bytes in 503 blocks
```

After:
```
definitely lost: 7,912 bytes in 352 blocks
indirectly lost: 27,535 bytes in 447 blocks
```

Evil villains which still leak a lot or in many places seem to be `LinkedListTest`, `HashDictionaryTest` and some geometry covers.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @sebastianbaginski ?

File: test/base/DebugTest.ooc [hunk 1]
```diff
@@ -46,6 +46,9 @@ DebugTest: class extends Fixture {
 			}
 		})
 	}
+	free: override func {
+		this outputString free()
```
Comment (sebastianbaginski) [inline test/base/DebugTest.ooc]: `super()`

**Comment by sebastianbaginski** [comment]
Nice.

File: test/base/DebugTest.ooc (hunk 1 above)
Comment (marcusnaslund) [inline test/base/DebugTest.ooc]: That's just super of you

**Pull request merged**